v2/config/samples/storage/v1alpha1api20210401_storageaccountsmanagementpolicy.yaml

```diff
@@ -1,26 +0,0 @@
-apiVersion: storage.azure.com/v1alpha1api20210401
```

Comment:
Deleted this file as this resource was not supported in `alpha` versions

Reply:
I brought this back, so I think you'll need to un-delete this now